src/devices/bus/epson_qx/option.cpp

```diff
@@ -6,8 +6,11 @@
  *
  *******************************************************************/
 #include "emu.h"
-#include "option.h"
+
+#include "cr1510.h"
+#include "ide.h"
 #include "multifont.h"
+#include "option.h"
```

Comment:
The corresponding header for the translation unit really should come immediately after the prefix header to avoid weird dependency issues.